src/content/contact_us.json

```diff
@@ -0,0 +1,18 @@
+{
+    "sections": [
+        {
+            "title": "Contact Us",
+            "content": [
+                {
+                    "par": "Have a question or want to suggest improvements for our project? We can help."
+                },
+                {
+                    "par": "Head to AnitaB.org Open Source Zulip Chat @https://anitab-org.zulipchat.com and get your doubts cleared in #questions stream"

```

Comment:
I think its head over to or you can write go to, wouldn't that be better?

Reply:
Yes it would me. Changing it!

Reply:
Done 👍